### PR TITLE
SIMSBIOHUB-537: restrict user options till searchText is > 2

### PR DESCRIPTION
**Pull request by KjartanE**
## Links to Jira Tickets

- https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-537?

## Description of Changes
- restrict user options till searchText is > 2

## Testing Notes

- {List any relevant testing considerations, necessary pre-reqs, and areas of the app to focus on. Specifically, include anything that will help the reviewers of this PR verify the code is functioning as expected.}

### CODE REVIEW

File: app/src/features/surveys/components/SurveyUserForm.tsx [hunk 1]
```diff
@@ -107,6 +107,7 @@ const SurveyUserForm = (props: ISurveyUserFormProps) => {
   };
 
   if (!searchUserDataLoader.data || !searchUserDataLoader.hasLoaded) {
+    console.log('searchUserDataLoader.data', searchUserDataLoader.data);
```
Comment (curtisupshall) [inline app/src/features/surveys/components/SurveyUserForm.tsx]: Remove

**Review by curtisupshall — COMMENTED** [review]
Tested locally, looks good!

**Review by NickPhura — DISMISSED** [review]

File: app/src/features/surveys/components/SurveyUserForm.tsx (hunk 1 above)
Comment (NickPhura) [reply app/src/features/surveys/components/SurveyUserForm.tsx]: Removed!

**Review by NickPhura — APPROVED** [review]
🐀

**Review by al-rosenthal — APPROVED** [review]
ran locally and looks good 🫕

**Pull request merged**